src/pages/staff.tsx

```diff
@@ -0,0 +1,85 @@
+                    job_title
+                    email
+                    phone
+                    headshot {
+                        ...SanityHeadshot
+                    }
+                }
+            }
+        }
+    `)
+
+    const allStaff = data.staff!.nodes!.map(person => {
+        return (
+            <div className="person">
+                <div className="photo">
+                    <Img fluid={person.headshot!.asset!.fluid}/>
+                </div>
+                <div className="info">
+                    <div className="name">{person?.name ?? "Missing name"}</div>
+                    <div className="title">{person?.job_title ?? "No job title"}</div>
+                    <div className="email"><a href={`mailto:${person?.email ?? "No email"}`}>{person?.email ?? "No email"}</a></div>
+                    <div className="bio">
+                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam iaculis risus sit amet ante congue, ut dignissim libero gravida. Duis et diam ut mi pharetra ornare eu ac felis. Morbi urna turpis, cursus nec tincidunt non, facilisis non libero. Vestibulum non ante id mauris tempus elementum. Integer ac tempus nulla. Curabitur porta tortor ac elit vestibulum, vel tempus risus sagittis. Fusce vitae sapien id leo tempor volutpat ut quis metus. Nunc id dui eu enim molestie euismod sit amet hendrerit erat. Nam ullamcorper eros in mi mollis, a viverra mauris scelerisque. Nunc volutpat maximus mi finibus consequat.</p>
+                        <p>Suspendisse tincidunt mattis ligula. Aliquam vestibulum urna ac vehicula porta. Nulla venenatis malesuada sapien, in tristique magna pharetra at. Vivamus lacinia neque eget mauris molestie, sit amet scelerisque diam condimentum. Suspendisse eu placerat elit. Proin interdum, sem eget aliquet vestibulum, ligula neque ullamcorper dui, eu dignissim diam lorem sit amet neque. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Sed sed arcu volutpat, blandit mi sed, facilisis odio.</p>
+                    </div>
+                </div>
+            </div>
+        )
+    })
+
+    return (
+        <Layout title={data.mainInfo?.frontmatter?.title} headerColour="dark">
+            <section className="header-underlay"></section>
+            <section className="intro wider dark">
+                <div
+                    className="text"
+                    dangerouslySetInnerHTML={{
+                        __html: data.mainInfo?.html ?? "No Content!",
+                    }}
+                />
+            </section>
+
+            <section className="staff">
+                <div>
+                    {allStaff}
+                </div>
+            </section>
+
+        </Layout>
+    )
+}
+
+export default IndexPage
+
+// I wish gatsby gave us a way to specify this in the config file or something. For now, let's keep it here.
+export const globallyReusableFragments = graphql`
```

Comment:
We can delete this whole bit

Reply:
I changed this to `StaffPage` but I'm not sure what you're suggesting otherwise.

Reply:
Oh sorry the comment was attached to the `globallyExportableFragments` chunk rather than `export default IndexPage`.